docs/resources/log.config

```diff
@@ -1,27 +1,27 @@
 [loggers]
-keys=root,modelLogger
+keys=root
 
 [handlers]
-keys=consoleHandler
+keys=consoleHandler,errorFileHandler
 
 [formatters]
 keys=fullFormatter
 
 [logger_root]
 level=INFO
-handlers=consoleHandler
-
-[logger_modelLogger]
-level=DEBUG
-handlers=consoleHandler
-qualname=modelLogger
-propagate=0
+handlers=consoleHandler,errorFileHandler
 
 [handler_consoleHandler]
 class=StreamHandler
 level=DEBUG
 formatter=fullFormatter
 args=(sys.stdout,)
 
+[handler_errorFileHandler]
```

Comment:
Just to clarify, the reason we are using the template approach with errorFileHandler rather than always adding it in add_file_log_handler() is because we want to have the option to disable the error log, correct?

Reply:
Right, also Yan mentioned that we wanted to have the ability to customize it, for example if he wanted it to be at INFO level or with a different formatter.